app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,42 @@
+class RelationshipsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :obtain_relationship, only: %i[update destroy]
+
+  def create
+    @relationship = current_user.active_relationships.build(relationship_params)
```

Comment:
I don't see `@relationship` variable in the views, however, I see a lot `current_user.active_relationships`
It makes sense to add `@active_relationships = current_user.active_relationships` instead of `@relationship`

Reply:
You are correct. I don't use @relationship in views, so I can change it to relationship.
I can add @active_relationships here, but I need to do the same in users#show.

Reply:
Is it a problem to add it to `users#show`?